### PR TITLE
Fixes issue #9027 - Instruction highlighter not updating

### PR DESCRIPTION
**Pull request by Maktm**
Fix for https://github.com/radare/radare2/issues/9027

### CODE REVIEW

File: libr/core/panels.c [hunk 1]
```diff
@@ -832,6 +832,10 @@ R_API int r_core_visual_panels(RCore *core) {
 		core->vmode = false;
 		r_core_visual_prompt_input (core);
 		core->vmode = true;
+
+		// FIX: Issue with visual mode instruction highlighter
+		// not updating after 'ds' or 'dcu' commands.
+		r_core_cmd0(core, ".dr*");
```
Comment (radare) [inline libr/core/panels.c]: missing space before (

Comment (Maktm) [reply libr/core/panels.c]: Do I make a new pull request?

**Comment by radare** [comment]
no, just commit here

**Comment by Maktm** [comment]
Alright, that fixes it

**Pull request merged**